lib/metadata/models/object/object.js

```diff
@@ -0,0 +1,465 @@
+
+// Version 2 changes the format of the location property
+const modelVersion = 2;
+
+/**
+ * Class to manage metadata object for regular s3 objects
```

Comment:
please update the comment so it is clear you are distinguishing between the metadata for an mpu part